components/css/button.css

```diff
@@ -0,0 +1,148 @@
+.btn {
+  border-radius: 4px;
+  color: var(--btn-primary-text);
+  transition: 0.3s;
+  opacity: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.btn_lg {
+  padding: 1.2rem 2rem;
+}
+
+.btn_sm {
+  padding: 8px 1rem;
+}
+
+.btn_primary {
+  background-color: var(--primary-color);
+}
+
+.btn_primary:hover,
+.btn_secondary:hover,
+.btn_success:hover,
+.btn_danger:hover,
+.btn_warning:hover,
+.btn_info:hover,
+.btn_floating:hover,
+.btn:hover {
+  opacity: 0.8;
+}
+
+.btn_secondary {
+  background-color: var(--secondary-btn);
+}
+
+.btn_outline {
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.btn_outline:hover {
+  background-color: var(--primary-color);
+  color: var(--btn-primary-text);
+}
+
+.btn_success {
+  background-color: var(--success-btn);
+}
+
+.btn_danger {
+  background-color: var(--danger-btn);
+}
+
+.btn_warning {
+  background-color: var(--warning-btn);
+  color: var(--btn-black-text);
+}
+
+.btn_info {
+  background-color: var(--info-btn);
+  color: var(--btn-black-text);
+}
+
+.btn.btn_disabled {
+  opacity: 0.6;
+  cursor: not-allowed;
+}
+
+.btn_link {
+  border-bottom: 1px solid var(--primary-color);
+}
+
+.btn_link:hover,
+.btn_icon:hover {
+  color: var(--primary-color);
+}
+
+.btn_floating {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--primary-color);
+  color: var(--btn-primary-text);
+  height: 3rem;
+  width: 3rem;
+  border-radius: 50%;
+}
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 3.75rem;
+  height: 2rem;
+}
+
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--toggle-background);
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
```

Comment:
try using rems

Reply:
yes, but since the fix was going on increasing, I decided to do that in another fix.